driver-pravega/pom.xml

```diff
@@ -62,19 +62,26 @@
         <dependency>
             <groupId>io.pravega</groupId>
             <artifactId>pravega-keycloak-client</artifactId>
-            <version>0.8.0</version>
+            <version>0.9.0</version>
```

Comment:
Can we use `0.9.0`?

Reply:
So, pravega-keycloak-client **0.9.0** was added in maven repo in March 2021.
I tested OMB with pravega-keycloak 0.9.0 and Pravega 0.9.1-2786.8992c07b9 with TLS. 
It works.